algebra-kernels/src/fft.rs

```diff
@@ -0,0 +1,191 @@
+use algebra_cl_gen::gpu::{GPUError, GPUResult, kernel_fft};
```

Comment:
Maybe it's worth to have a couple of unit tests asserting equality between the GPU result and the CPU one ?

Reply:
Inside the ginger-lib we have now "fft_consistency" test to check the results

Reply:
I would've preferred two separate tests but it's fine